Homework-№4-from-28.09.2022/task3.c

```diff
@@ -0,0 +1,205 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
+
+#define maxSizeOfEntry 101
```

Comment:
```suggestion
#define maxSizeOfEntry 100
```
Дополнительный элемент в массиве нужен только для строк, чтобы в нём хранился символ конца строки `\0`. Для остальных массивов это не нужно.